source/query/atlas-search.txt

```diff
@@ -25,32 +39,28 @@ Steps
    :style: normal
 
    .. step:: Find the index you want to use in your query
-      
-      a. From the :guilabel:`Indexes` tab, click :guilabel:`Search Indexes`.
-      #. Hover over the index you want to use in your query.
 
-   .. step:: Run the query
+      From the :guilabel:`Indexes` tab, click :guilabel:`Search Indexes`. 
+      Hover over the search index you want to use in your query. 
       
-      a. Click the :guilabel:`Aggregate` button.
-
-         |compass| switches to the :guilabel:`Aggregations` tab where you 
-         can run your query. 
-        
-      #. Construct your query.
+   .. step:: Open the aggregation pipeline builder
+      
+      Click the :guilabel:`Aggregate` button that appears when you hover over 
+      your search index.
          
-         By default, |compass| selects the :pipeline:`$search` stage for
-         your query. You can select the  :pipeline:`$searchMeta` stage
-         to use in your query. If you are using a ``vectorSearch`` type
-         index in your query, you can only use the
-         :pipeline:`$vectorSearch` stage in your query.  
+   .. step:: Construct your query
+      
+      By default, |compass| selects the :pipeline:`$search` stage for your 
+      query. You can select the  :pipeline:`$searchMeta` stage to use in your 
+      query. If you are using a ``vectorSearch`` type index in your query, you 
+      can only use the :pipeline:`$vectorSearch` stage in your query. 
 
-         .. note:: 
+      .. note:: 
 
-            The Atlas Search :pipeline:`$search` and :pipeline:`$searchMeta` and 
-            the Atlas Vector Search :pipeline:`$vectorSearch` pipeline stages must 
-            be the first stage in your aggregation pipeline.
+         If you use the ``$search``, ``$searchMeta``, or ``$vectorSearch`` 
+         stage, it must be the first stage in your aggregation pipeline.
```

Comment:
This is mostly true, but hybrid search is an exception when both vector and full-text search are used in the same pipeline: https://www.mongodb.com/docs/atlas/atlas-search/tutorial/hybrid-search/

Is this worth calling out here?

Reply:
Filed a followup: https://jira.mongodb.org/browse/DOCSP-46151